mojoshader_common.c

```diff
@@ -1135,6 +1135,8 @@ void MOJOSHADER_spirv_link_attributes(const MOJOSHADER_parseData *vertex,
     {
         if (vTable->attrib_offsets[MOJOSHADER_USAGE_POINTSIZE][0] > 0)
         {
+            ((uint32 *) pixel->output)[pTable->pointcoord_var_offset + 1] = pTable->tid_pvec2i;
+            ((uint32 *) pixel->output)[pTable->pointcoord_load_offset + 1] = pTable->tid_vec2;
```

Comment:
We'll need a reverse version of this for the `else` case - essentially, setting these to tid_pvec4i and tid_vec4 when it's not in point mode.

Reply:
Good point. It's been a while and for some reason I thought we were patching temporary copy of the binary, so I relied on the input to always contain texcoord0 case. This would have been an unpleasant bug to track down.